tools/accuracy_checker/accuracy_checker/postprocessor/resize_style_transfer.py

```diff
@@ -0,0 +1,46 @@
+import numpy as np
+from ..postprocessor import Postprocessor
+from ..representation import StyleTransferAnnotation, StyleTransferPrediction
+from ..config import NumberField
+from ..utils import get_size_from_config
+
+try:
+    from PIL import Image
+except ImportError:
+    Image = None
+
+
+class ResizeStyleTransfer(Postprocessor):
+    __provider__ = 'resize_style_transfer'
+
+    annotation_types = (StyleTransferAnnotation, )
+    prediction_types = (StyleTransferPrediction, )
+
+    @classmethod
+    def parameters(cls):
+        parameters = super().parameters()
+        parameters.update({
+            'dst_width': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination width for resizing."
+            ),
+            'dst_height': NumberField(
+                value_type=int, optional=True, min_value=1, description="Destination height for resizing."
+            ),
+            'size': NumberField(
```

Comment:
Seems to me, this attribute is unused. Please, delete or implement it

Reply:
Fixed in https://github.com/opencv/open_model_zoo/pull/1106/commits/b78761cab1327b54ae9b3770ff81dbc1f0bee8f6